### ISSUE TITLE
Spider: Chicago Special Service Area #20 South Western Avenue

### ISSUE THREAD

**Issue by pjsier** [City-Bureau/city-scrapers#880]
URL: https://www.mpbhba.org/business-resources/
Spider Name: chi_ssa_20
Agency Name: Chicago Special Service Area #20 South Western Avenue

See the [contribution guide](https://cityscrapers.org/docs/development/#contribute) for information on how to get started

**Comment by pg** [comment]
It seems there are no events under https://www.mpbhba.org/events/

**Comment by pjsier** [comment]
Yep, all of the events are on https://www.mpbhba.org/business-resources/ in the description down the page

**Comment by Sidray-Infinity** [comment]
Trying this as my first contribution.

**Comment by benjaminfawcett** [comment]
I'm interested in taking a stab at this one if no one else is actively working on it

**Comment by rajat641** [comment]
I'm interested in working on this.

**Comment by jmelot** [comment]
I was going to volunteer for this one, but it looks like the last meeting posted was in July 2019. Would it be worth contacting them to see if they're planning to make updates in the future, or better to just hold off and see if they start updating again?

**Comment by pjsier** [comment]
@jmelot I think we've typically just been writing scrapers even without updates because then we have the updates as soon as they're made, but you're welcome to work on that other one if you're interested!

**Comment by guytet** [comment]
Hi,
I'm trying to work on this one. Could not find a `chi_ssa_20` under `city_scrapers/spiders/` so I assumed the issue is up for grabs (also, no `claimed` label. 
I sent a request to join your slack channel.

**Comment by pjsier** [comment]
Sure! All yours @guytet
